app/assets/javascripts/modal/modal-workflow.js

```diff
@@ -34,6 +35,14 @@ ModalWorkflow.prototype.renderError = function (result) {
   window.Raven.captureException(result)
   console.error(result)
   this.$multiSectionViewer.showStaticSection('error')
+  this.setModalLabel()
+}
+
+ModalWorkflow.prototype.setModalLabel = function () {
```

Comment:
I'd also be tempted to have this take a parameter:

```suggestion
ModalWorkflow.prototype.setModalLabel = function (ariaLabel) {
```

It would bring the `setModalLabel` calls into line with the `show(Static|Dynamic)Section` calls, insofar as they take a parameter, making it less stateful and more obvious what it's doing. Would end up looking something like `this.setModalLabel(this.getSectionHeading())`.

Reply:
Thanks for the suggestion, Chris. It makes sense to aim for stateless, will update the code.